### PR TITLE
parser documentation

### PR DESCRIPTION
**Pull request by fgoudreault**
- Made some docstrings and placed comments in the source code.

### CODE REVIEW

File: emva1288/process/parser.py [hunk 1]
```diff
@@ -76,9 +90,11 @@ def _get_images_filenames(self):
         return fnames
 
     def _get_kind(self, fnames):
-        '''
+        """
         Guess what kind of data based on the number of images
-        '''
+        Temporal = 2 images for each measurement point
```
Comment (fariza) [inline emva1288/process/parser.py]: argument?

File: emva1288/process/parser.py [hunk 2]
```diff
@@ -88,41 +104,64 @@ def _get_kind(self, fnames):
         return kind
 
     def _add_pcount(self, exposure, photons, fnames):
-        '''Add images to a given exposure/phton
+        """Add images to a given exposure/phton
```
Comment (fariza) [inline emva1288/process/parser.py]: arguments?

File: emva1288/process/parser.py [hunk 3]
```diff
@@ -132,45 +171,57 @@ def _fill_info(self):
...
                 self._add_pcount(exposure, np.float(0.0), fnames)
 
                 continue
 
+            # If line is of an unknown format, warn user.
             self.log.warning('Unknown command ' + line)
 
     def _split_line(self, line):
-        '''
+        """
```
Comment (fariza) [inline emva1288/process/parser.py]: argument?

File: emva1288/process/parser.py [hunk 4]
```diff
@@ -132,45 +171,57 @@ def _fill_info(self):
...
         l = [x.strip() for x in line.split()]
         if (not l) or (len(l) < 2):
             raise SyntaxError('Wrong format line: %s' % line)
         return l
 
     def _load_file(self, filename):
-        '''
+        """
```
Comment (fariza) [inline emva1288/process/parser.py]: argument?

File: emva1288/process/parser.py (hunk 1 above)
Comment (fgoudreault) [inline emva1288/process/parser.py]: https://www.python.org/dev/peps/pep-0008/#string-quotes

File: emva1288/process/parser.py (hunk 2 above)
Comment (fgoudreault) [inline emva1288/process/parser.py]: https://www.python.org/dev/peps/pep-0008/#string-quotes

File: emva1288/process/parser.py (hunk 3 above)
Comment (fgoudreault) [inline emva1288/process/parser.py]: https://www.python.org/dev/peps/pep-0008/#string-quotes

File: emva1288/process/parser.py (hunk 4 above)
Comment (fgoudreault) [inline emva1288/process/parser.py]: https://www.python.org/dev/peps/pep-0008/#string-quotes

**Pull request merged**